File: Student.cpp
#include<iostream>
using namespace std;
class Student
{
public:
string fullName;
int rollNum;
double semPerentage;
string collegeName;
int collegeCode;
Student()
{
cout<<"Student class initialized!!\nIam a student\n";
}
~Student(){}

};
int main()
{
Student meghana;
cout<<"Enter your name : ";
getline(cin,meghana.fullName);
cout<<"Enter your roll number : ";
cin>>meghana.rollNum;
cout<<"Enter your sem percentage : ";
cin>>meghana.semPerentage;
cout<<"Enter your college name : ";
cin>>meghana.collegeName;
cout<<"Enter your college code : ";
cin>>meghana.collegeCode;
cout<<"Name : "<<meghana.fullName<<endl;
cout<<"RollNum : "<<meghana.rollNum<<endl;
cout<<"SemPercentage : "<<meghana.semPerentage<<endl;
cout<<"CollegeName : "<<meghana.collegeName<<endl;
cout<<"CollegeCode : "<<meghana.collegeCode<<endl;
return 0;
}

File: Student.java
import java.util.Scanner;
class Student508
{
public
String fullName;
int rollNum;
double semPercentage; 
String collegeName;
int collegeCode;
Student508()
{
System.out.println("Student class is created successfully!!\nIam a student\n");
}
protected void finalize() throws Throwable
{
System.out.println("The class is finalized\n");
}
}
public class Student
{
public static void main(String[] args)
{
Student508 meghana=new Student508();;
Scanner input=new Scanner(System.in);
System.out.print("Enter your  name : ");
meghana.fullName=input.nextLine();
System.out.print("Enter your rollNumber : ");
meghana.rollNum=input.nextInt();
System.out.print("Enter your semPercentage : ");
meghana.semPercentage=input.nextDouble();
input.nextLine();
System.out.print("Enter your collegeName : ");
meghana.collegeName=input.nextLine();
System.out.print("Enter your collegeCode : ");
meghana.collegeCode=input.nextInt();
System.out.print("Name : "+meghana.fullName+"\n");
System.out.print("RollNumber : "+meghana.rollNum+"\n");
System.out.print("SemPercentage : "+meghana.semPercentage+"\n");
System.out.print("CollegeName : "+meghana.collegeName+"\n");
System.out.print("CollegeCode : "+meghana.collegeCode+"\n");
input.close();
}
}
